usecases/case_usecase.go

```diff
@@ -152,7 +152,7 @@ func (usecase *CaseUseCase) GetCase(ctx context.Context, caseId string) (models.
 		return models.Case{}, err
 	}
 
-	availableInboxIds, err := usecase.getAvailableInboxIds(ctx, exec)
+	availableInboxIds, err := usecase.getAvailableInboxIds(ctx, exec, c.OrganizationId)
```

Comment:
I may need to review this - tomorrow

Reply:
Ok, I had a doubt about using the orgId from the case itself to get the available inbox ids and then do the permission check based on that. 
After verification, it's ok because getAvailableInboxIds calls inboxReader.ListInboxes which itself enforces the permission check on the inboxes it returns.